enterprise-suite/es-grafana/jvm-exporter.json

```diff
@@ -75,37 +75,35 @@
     "graphName": "GC Rate",
     "promQL": [
       {
-        "expr": "sum by (kubernetes_pod_name)(rate(jvm_PS_MarkSweep_count{ContextTags}[5m]))",
-        "legendFormat": "PS_MarkSweep Rate {{kubernetes_pod_name}}"
-      },
-      {
-        "expr": "sum by (kubernetes_pod_name)(rate(jvm_PS_Scavenge_count{ContextTags}[5m]))",
-        "legendFormat": "PS_Scavenge Rate {{kubernetes_pod_name}}"
+        "expr": "rate( (jvm_gc_collection_seconds_count{ContextTags} or on (kubernetes_pod_name, gc) label_replace({ContextTags,__name__=~\"jvm_.*_count\",gc=\"\"}, \"gc\", \"$1\", \"__name__\", \"jvm_(.*)_count\")) [5m:])",
+        "legendFormat": "{{gc}} rate {{kubernetes_pod_name}}"
       }
-    ]
+    ],
+    "yaxes": {
+      "format": "ops"
+    }
   },
   {
     "graphName": "GC Time",
     "promQL": [
       {
-        "expr": "max by (kubernetes_pod_name)(jvm_PS_MarkSweep_time{ContextTags})",
-        "legendFormat": "PS_MarkSweep max {{kubernetes_pod_name}}"
-      },
-      {
-        "expr": "max by (kubernetes_pod_name)(jvm_PS_Scavenge_time{ContextTags})",
-        "legendFormat": "PS_Scavenge max {{kubernetes_pod_name}}"
+        "expr": "rate( (jvm_gc_collection_seconds_sum{ContextTags} or on (kubernetes_pod_name, gc) label_replace({ContextTags,__name__=~\"jvm_.*_time\",gc=\"\"}, \"gc\", \"$1\", \"__name__\", \"jvm_(.*)_time\") /1000) [5m:])",
+        "legendFormat": "{{gc}} time {{kubernetes_pod_name}}"
```

Comment:
Same comment as before... using `rate` effectively smooths out the GC time, and will make it look much better than it really is. The important aspect of GC time is seeing if you have very long pause times. This will be lost. At minimum, the title of the graph should reflect this is an "average per second".

Reply:
graph title changed